path-animate/demo.html

```diff
@@ -117,7 +117,45 @@
                         .drawCircle(0, 0, Math.max(this.width, this.height) * 0.6);
                     highlight.updateCache();
                 }
-            }
+            };
+
+            airplane.setPosition = function () {
+                var temp = Math.floor(Math.random() * 4); // determines which side airplane will enter from
+                var endpoint = [];
```

Comment:
using a array to store x and y position is not a good use of array or position. try to use an already accepted way to keep track of position. in the rest of the code, we with use `pointX` and `pointY` as separate variables (like stage, mouse), or we use a point object `point.x, point.y`. either of these would be better, and self-documenting.